Components/Sources/Components/ShimmerView/ShimmerConfig.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  [NAME].swift
+//  StackOv ([NAME] module)
```

Comment:
Unfortunately, Xcode cannot put informations automatically in SPM packages yet 🙃